### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by pasha30061**
- [DEMO LINK](https://pasha30061.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://pasha30061.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,46 @@
...
+          </li>
+          <li class="nav_item">
+            <a href="#samsung" class="nav_link">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a href="#smartphones" class="nav_link">Smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a href="#laptops & computers" class="nav_link"
+            data-qa="hover">Laptops & Computers</a>
```
Comment (Viktor-Kost) [inline src/index.html:33]:
```suggestion
            <a 
              href="#laptops&computers" 
              class="nav_link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,67 @@
...
+.nav_item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav_link {
+  color: #000;
+  display: flex;
+  white-space: nowrap;
+  align-items: center;
+  height: 60px;
```
Comment (Viktor-Kost) [inline src/style.css:42]: it's not a good practice to hardcode height for elements with content inside. just use padding

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,67 @@
...
+.nav_link:hover {
+  color: var(--color);
+}
+
+a.is-active {
+  color: var(--color);
+  position: relative;
+}
+
+a.is-active::after {
```
Comment (Viktor-Kost) [inline src/style.css:56]: use class selectors instead of tag ones

File: src/index.html [hunk 4]
```diff
@@ -6,9 +6,46 @@
...
-    <h1>Moyo header</h1>
+    <header id="home" class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="Moyo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#apple" class="nav_link is-active">Apple</a>
```
Comment (Viktor-Kost) [inline src/index.html:23]: according to BEM methodology
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
```

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,67 @@
...
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav {
+  display: block;
+}
```
Comment (Viktor-Kost) [inline src/style.css:18]: this tag is a block by default
```suggestion
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
GJ!
Some minor fixes needed

**Comment by pasha30061** [comment]
[corrected the details]
[DEMO LINK](https://pasha30061.github.io/layout_moyo-header/)
[TEST REPORT LINK](https://pasha30061.github.io/layout_moyo-header/report/html_report/)

File: src/index.html [hunk 6]
```diff
@@ -6,9 +6,51 @@
...
+          </li>
+          <li class="nav__item">
+            <a
+            href="#laptops & computers"
+            class="nav__link"
+            data-qa="hover"
+            >
+            Laptops & Computers
+            </a>
+          </li>
```
Comment (etojeDenys) [inline src/index.html:39]:
```suggestion
            <a
              href="#laptops & computers"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
          </li>
```

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Comment by pasha30061** [comment]
add new commit

**Review by etojeDenys — APPROVED** [review]
good job

**Pull request closed without merging**